grails-app/views/project/_projectAdmin.gsp

```diff
@@ -70,12 +74,16 @@
                           <g:render template="riskTable" model="[project:project]"/>
                       </div>
                   </div>
-                  <div id="risks-reporting-section" class="pill-pane tab-pane">
-                      <g:render template="riskReporting" model="[project:project]"/>
-                  </div>
                 </g:if>
 
             </g:if>
+            <g:if test="${user.isAdmin || user.isCaseManager || fc.userHasReadOnlyAccess() }">
```

Comment:
As per the other review, please update the controller so the canViewRisks works as desired rather than add an extra check here.